content/app-dev/create-actor/update.md

```diff
@@ -5,6 +5,29 @@ weight: 5
 draft: false
 ---
 
+### Setting up Hot Reloading
+
+This section will introduce you to the process wasmCloud developers use to rapidly iterate on their actors. Introduced in wasmCloud `v0.50.4`, the Hot Reloading feature allows the dashboard to watch your signed actor file and automatically replace all instances of that actor upon a file change. This happens quickly (blink and you might miss it!) and tightly constrains the developer feedback loop so you can spend more time developing and less time waiting.
+
+Let's start by scaling down our `hello` actor if it's still running from a previous step. Look for the yellow scale button <img src="../scalebutton.png" alt="scalebutton" width="30px"/> in the 'Actions' column, and set the replicas to `0` before hitting Submit.
+
+![scaledown](../scaledown.png)
+
+Now, let's start the actor again using the "Start Actor" dropdown, but this time let's use the _From File (Hot Reload)_ option. You'll be required to supply the absolute path [^1] to your actor file, and you can choose a number of replicas to start as well. Luckily, we have an easy way for you to get the absolute path of your actor. Simply navigate to your actor project directory and run:
+```shell
+make target-path-abs
+```
+
+Copy this file location and input it in the _Start Actor_ form. You can start multiple replicas of the actor here, I've chosen to start with `15` so that I have a few replicas while I test.
```

Comment:
Copy the path printed by this command into the

Reply:
Did you mean to include the sentence "I've chosen to start with ... ."

Reply:
I did, though I can see that it's a little clunky. I was trying to indicate the number of replicas in the screenshot and let the user choose an amount that they want to start